RecoTracker/MkFitCore/standalone/xeon_scripts/init-env.sh

```diff
@@ -1,9 +1,16 @@
 #!/bin/bash
-if [[ `lsb_release -si` == "Fedora" ]]
+if [[ `[ -f /usr/bin/lsb_release ] && lsb_release -si` == \
+        "Fedora" ||
+      `[ -f /etc/redhat-release ] && awk '{print $1}' /etc/redhat-release` == \
+        "AlmaLinux" ]]
 then
-  source /cvmfs/cms.cern.ch/el8_amd64_gcc10/lcg/root/6.24.07-da610b2b7ed663a0a05d3605f3d83ceb/etc/profile.d/init.sh
-  export TBB_GCC=/cvmfs/cms.cern.ch/el8_amd64_gcc10/external/tbb/v2021.5.0-e966a5acb1e4d5fd7605074bafbb079c/
+  source /cvmfs/cms.cern.ch/el8_amd64_gcc11/lcg/root/6.26.07-e76d33b57cbf5a1c12b870df2a6ebb79/etc/profile.d/init.sh
+  export TBB_GCC=/cvmfs/cms.cern.ch/el8_amd64_gcc11/external/tbb/v2021.8.0-791ebc4967ab49af60ca9ad2aa021259
+#  source /cvmfs/cms.cern.ch/el8_amd64_gcc10/lcg/root/6.24.07-3ea108fb48fc7b8bb2960269f724c0d6/etc/profile.d/init.sh
+#  export TBB_GCC=/cvmfs/cms.cern.ch/el8_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17
```

Comment:
commented out code without at least some note why it's here often triggers review questions.
Please clarify why these are kept.
BTW, considering that only the SCRAM_ARCH part changes, perhaps the script can even be with if/else only for setting `el8_amd64_gcc11` vs `slc7_amd64_gcc11`

Reply:
@slava77 I wasn't sure if the approved level of gcc for standalone testing would be gcc-10 or gcc-11. Happy to remove the commented code if you agree that there is no problem with moving to gcc-11.

And yes, I considered whether the if-else block should just set a script variable `os_arch_comp` to be either `el8_amd64_gcc11` or (if the tests fail)`slc7_amd64_gcc11`; after which, the value of `os_arch_comp` is inserted into the paths for ROOT and TBB, which are otherwise identical, as you point out. This would probably be a clearer way of doing things, so I'm willing to make that change as well.

If `os_arch_comp` happens to be equivalent to `SCRAM_ARCH` in the CMS build system (which I don't use), then I suppose I could add a test at the top to set `os_arch_comp` to the value of `SCRAM_ARCH` if it exists in the env, otherwise set it to the result of the existing if-else, and proceed. Would you prefer that? I think it's not guaranteed that the particular hashes that I identified as being valid for ROOT and TBB exist for all values of `SCRAM_ARCH`.

Reply:
> @slava77 I wasn't sure if the approved level of gcc for standalone testing would be gcc-10 or gcc-11. Happy to remove the commented code if you agree that there is no problem with moving to gcc-11.

@makortel please advise if there are reasons to keep gcc-10 setup alive even as a backup.

Reply:
I don't think CMS would revert to gcc 10 (we have been using gcc 11 as production compiler for ~3 months and have not seen any issues).

Reply:
Just pushed a commit to this branch making the changes indicated in my comment above: gcc-10 is removed, `os_arch_comp` is set to simplify the if-else, and no test for `SCRAM_ARCH`.